en/SUMMARY.md

```diff
@@ -115,38 +132,21 @@
   * [Log Analysis using PlotJuggler](log/plotjuggler_log_analysis.md)
 * [Advanced Configuration](advanced_config/README.md)
   * [Finding/Updating Parameters](advanced_config/parameters.md)
-  * [Multicopter Config/Tuning](config_mc/README.md)
-    * [MC Filter/Control Latency Tuning](config_mc/filter_tuning.md)
-    * [MC PID Tuning (Manual/Basic)](config_mc/pid_tuning_guide_multicopter_basic.md)
-    * [MC PID Tuning Guide (Manual/Advanced)](config_mc/pid_tuning_guide_multicopter.md)
-    * [MC Setpoint Tuning (Trajectory Generator)](config_mc/mc_trajectory_tuning.md)
-      * [MC Jerk-limited Type Trajectory](config_mc/mc_jerk_limited_type_trajectory.md)
-    * [Multicopter Racer Setup](config_mc/racer_setup.md)
-  * [Fixed Wing  Config/Tuning](config_fw/README.md)
-    * [Fixedwing PID Tuning Guide](config_fw/pid_tuning_guide_fixedwing.md)
-    * [Fixedwing Advanced Tuning Guide](config_fw/advanced_tuning_guide_fixedwing.md)
-    * [Fixedwing Trimming Guide](config_fw/trimming_guide_fixedwing.md)
-  * [VTOL Config/Tuning](config_vtol/README.md)
-    * [QuadPlane Configuration](config_vtol/vtol_quad_configuration.md)
-    * [Back-transition Tuning](config_vtol/vtol_back_transition_tuning.md)
-    * [VTOL w/o Airspeed Sensor](config_vtol/vtol_without_airspeed_sensor.md)
-    * [VTOL Weather Vane](config_vtol/vtol_weathervane.md)
-  * [ESC Calibration](advanced_config/esc_calibration.md)
+  * [Full Parameter Reference](advanced_config/parameter_reference.md)
   * [ECL/EKF Overview & Tuning](advanced_config/tuning_the_ecl_ekf.md)
   * [Flight Termination Configuration](advanced_config/flight_termination.md)
-  * [Bootloader Update](advanced_config/bootloader_update.md)
   * [Bootloader Flashing onto Betaflight Systems](advanced_config/bootloader_update_from_betaflight.md)
   * [Land Detector Configuration](advanced_config/land_detector.md)
+  * [Prearm/Arm/Disarm Configuration](advanced_config/prearm_arm_disarm.md)
```

Comment:
I wonder if a simplified version of this should go up in the safety setup in standard configuration (ideally in its own QGC screen). Essentially to control the settings of whether a safety button should be used if present, and whether prearm is the default equivalent of disarmed.

Reply:
Totally, many of these options would get very simple to use intuitively if the UI was up to speed 😬

Reply:
Do you have a fair idea of what you'd like the UI to look like? If we can draw it we can suggest it as an intern job or similar.

In the meantime, it might be worth having a docs section in safety setup for choosing the safety switch/arm behaviour - perhaps an extension of http://docs.px4.io/main/en/config/safety.html#arm-disarm-switch

Reply:
PS, thanks for all these comments. Very helpful. It is scary to make major changes to structure without the team being involved.